source/administration/changelog.md

```diff
@@ -4,6 +4,127 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
 
 Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.33 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
+
+**Release Day: 2021-03-16**
+
+### Breaking Changes
+ - Deleting a reaction is now a soft delete in the ``Reactions`` table. A schema update is required and may take up to 15 seconds on first run with large data sets.
```

Comment:
Not sure I understand the impact of this breaking change, why it exists, or what action items I have to take, or what happens if I don't do anything (from the perspective of a sys admin)

Reply:
@wiggin77 - please help share details here.

Reply:
I don't know what has been defined as a breaking change, but this is just a schema change and doesn't change how any APIs behave.  The need for a release notice is simply to warn the admin that the database migration may take longer than usual.